cloudify/workflows/events.py

```diff
@@ -48,7 +54,54 @@ def _send_task_event_func(task, event_type, message, out_func,
 
 
 def _filter_task(task, state):
-    return state != tasks_api.TASK_FAILED and not task.send_task_events
+    return state != TASK_FAILED and not task.send_task_events
+
+
+def get_event_type(state):
+    if state == TASK_SENDING:
```

Comment:
A dict might be better here- it'll be shorter as more things are added, and could be reused in other places if it needed to be.

Reply:
yeah, I thought about that. But then I decided I preferred this, so that it stays similar to the other function. wdyt?

Reply:
Assuming you mean format_event_message, I was looking at that and wondering if we can maybe do something better since both of these seem to have (aside from sending) very similar structures.

Reply:
e.g. here we could do something like:
if state == TASK_SENDING: return 'sending_task'
elif state in (TASK_STARTED, TASK_SUCCEEDED, TASK_RESCHEDULED, TASK_FAILED): return 'task_{}'.format(state)
else: raise...

The format_event_message could do something similar for the basic message formatting- message = "Task {} '{{0}}'".format(state)
That'll be a few reduced lines and make the special cases look more special?